### DISCUSSION TITLE
Oracle ATG

### DISCUSSION THREAD

**Discussion by arshan700** [community/community#36260] [Other Feature Feedback, Questions, & Ideas]
Can someone let me know how to get subscription of Oracle ATG

**Comment by queenofcorgis** [comment]
Hi @arshan700,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Oracle. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by arshan700** [comment]
Thanks for contacting me
  I ld like to contact and get the membership of Oracle OTG programme for
sams club and wallmart order refund.
let me know either you may help me out because i m finding the indirect
links and have to pay a lot of commission but the direct contact may reduce
my cost .
  Thanks for understanding !
   Regards
